InteractiveAutomationToolkit/InteractiveController.cs

```diff
@@ -155,6 +155,20 @@ public void Update()
 			CurrentDialog.Show(false);
 		}
 
+		/// <summary>
+		/// Hides current dialog.
```

Comment:
Can you provide some more details? E.g. "Hides the UI. This does not block any background logic from running. Use see cref="ShowDialog" (CANT FORMAT THIS IN GITHUB REVIEW) if you want to show the UI again."